boards/Kconfig

```diff
@@ -2045,6 +2045,7 @@ config ARCH_BOARD_RASPBERRYPI_PICO_W
 config ARCH_BOARD_RASPBERRYPI_4B
 	bool "Raspberry Pi Model 4B"
 	depends on ARCH_CHIP_BCM2711
+    select ARCH_HAVE_LEDS
```

Comment:
space to tab, cc @simbit18

Reply:
Whoops! Would be good if checkpatch could catch this. I have another PR that should be coming soon for SDMMC support; I can fix this issue there.